.github/workflows/certified_images.yml

```diff
@@ -0,0 +1,161 @@
+name: Certified Images
+
+on:
+  workflow_dispatch:
+    inputs:
+      java_major:
+        description: 'Java major version to use for all cloud providers'
+        required: true
```

Comment:
Is this really an input? Or is it a matrix axis?

Sample usage in [gatling-maven-plugin](https://github.com/gatling/gatling-maven-plugin/blob/main/.github/workflows/ci.yml#L14-L16)

Reply:
Hmm, agreed using a matrix will reduce the amount of operations to do in order to release!